lib/vrl/lookup/src/lookup_v2/mod.rs

```diff
@@ -1,3 +1,54 @@
+//! This crate contains all of the logic for lookup paths.
+//!
+//! # Traits
+//! There are 2 main traits that define a path.
+//! - [ValuePath] is a path that points inside of a `Value`
```

Comment:
I'm not sure "points inside of a `Value`" is clear - can that be expanded or phrased differently?

Reply:
Agreed. Even "Points to a `Value`", would be better. But "polnts to" seems like it could be improved on. (Same goes for the Target path).

edit- I'm seeing the phrase "point<> to" used below as well. Maybe that concept could be described in the opening section (before `# Traits`) ?

Reply:
I added some clarification / explanation at the top. I do like using the term "point" since I think of paths as quite literally pointers to some value. Let me know if it's still confusing.